<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CParamTableMngrWid</class>
 <widget class="QWidget" name="CParamTableMngrWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_fileNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名/序号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fileNo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bakeup">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>备份</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_restore">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_preview">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>预览当前参数表</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>651</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>651</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名/序号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
